build.gradle.kts

```diff
@@ -35,37 +35,37 @@ version = "0.0.1-SNAPSHOT"
 description = "KappaBot: a Discord chat bot"
```

Comment:
While you're in here, feel free to change this if you are looking at adding additional official chat integrations. Regardless, it seems like KappaBot has been written with the idea of being easily expandable to other services too so something more generic might be good.

Reply:
Updated. I also rebased onto master to pick up the other changes that were merged in.